index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
+  <link rel="stylesheet" href="./syles.css">
+  <title>Web Dev</title>
+</head>
+
+<body>
+  <ul class="noStyle" id="topnav">
+    <li class="topli"><a class="noAncStyle" href="#"><b>f</b></a></li>
+    <li class="topli"><a class="noAncStyle" href="#"><img id='toptweet' src="./assets/twitter.png"
+          alt="link to twitter"></a></li>
+    <li class="topli"><a class="noAncStyle" href="#">Swahili</a></li>
+    <li class="topli"><a class="noAncStyle" href="#">Invite</a></li>
+    <li class="topli" id="face"><a class="noAncStyle" href="#">Logout</a></li>
+  </ul>
+  <div id="firstback">
+    <header>
+      <nav>
+        <div class="ham-space">
+          <i class="fa fa-bars fa-3x" id="hum-menu" aria-hidden="true"></i>
+        </div>
+        <div class="blurBack hideMenu">
+          <div class="ham-space" id="cancel"><i class="fa fa-times fa-3x" aria-hidden="true"></i></div>
+          <a href="./index.html" class="noDisplay"><img id="logo" src="./assets/logo.jpg" alt="Company logo"></a>
+          <ul class="noStyle deskarr">
+            <li class="listStle"><a class="noAncStyle" href="./about.html">About</a></li>
+            <li class="listStle"><a class="noAncStyle" href="#">Program</a></li>
+            <li class="listStle"><a class="noAncStyle" href="#">join</a></li>
+            <li class="listStle"><a class="noAncStyle" href="#">Sponsor</a></li>
+            <li class="listStle"><a class="noAncStyle" href="#">News</a></li>
+          </ul>
+          <button id="Btn">CC Campaign</button>
+        </div>
+      </nav>
+    </header>
```

Comment:
- [ ] [REQUIRED] I really appreciate what you have done but I think you have a little issue in the `positioning` of the `navigation elements` because the positioning of the elements in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) is different from yours, please try to fix this issue 🙏 

### Your design ❌ 

![image](https://user-images.githubusercontent.com/63915024/203793377-68918be9-0508-48e2-a982-9e20bad5c369.png)

### [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) ✔️ 

![image](https://user-images.githubusercontent.com/63915024/203793535-f7d207fb-6197-438c-a735-f2abdb5d8f32.png)